tracer/tracer.go

```diff
@@ -61,13 +62,15 @@ type EpsagonLabel struct {
 type epsagonTracer struct {
 	Config *Config
 
-	eventsPipe     chan *protocol.Event
-	events         []*protocol.Event
-	exceptionsPipe chan *protocol.Exception
-	labelsPipe     chan EpsagonLabel
-	exceptions     []*protocol.Exception
-	labels         map[string]interface{}
-	labelsSize     int
+	eventsPipe          chan *protocol.Event
+	events              []*protocol.Event
+	runnerExceptionPipe chan *protocol.Exception
```

Comment:
can't we do it with the existing exception pipe?

Reply:
no because its used for another purpose (our internal exceptions, not a custom one)